src/libs/ReportUtils.js

```diff
@@ -681,6 +681,45 @@ function getSmallSizeAvatar(avatarURL, login) {
     return `${source.substring(0, lastPeriodIndex)}_128${source.substring(lastPeriodIndex)}`;
 }
 
+/**
+ * Returns the appropriate icons for the given chat report using the stored personalDetails.
+ * The Avatar sources can be URLs or Icon components according to the chat type.
+ *
+ * @param {Array<*>} participants
```

Comment:
is there a benefit/difference of doing `{Array<*>}` vs `{Array}`?

Reply:
Hmm I'm not sure

Reply:
let's get rid of it i don't think we really do that anywhere else

Reply:
cool, thanks for the thoughts.